types/koa-joi-router-docs/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for koa-joi-router-docs 1.0
+// Project: https://github.com/o2team/koa-joi-router-docs#readme
+// Definitions by: 4doge <https://github.com/4doge>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { Router } from 'koa-joi-router';
+
+export {};
+
+declare namespace koaJoiRouterDocs {
```

Comment:
Why the namespace? The file is already private and the interfaces used in the public API should _probably_ be enabled regardless.